### PR TITLE
Add logging to Splunk for My Account URL and general errors

### PR DESCRIPTION
**Pull request by arthurepc**
#### What is the purpose of this pull request?

Title is self-explanatory.

#### What problem is this solving?

Log accounts that use `/account/orders` in its URL in order to map them. Besides, it adds general error logging.

#### How should this be manually tested?

go to [1] and access My Account. Then, go to [2] and check that your access was logged.

[1]: https://cobasi.myvtex.com/_secure/account/orders?workspace=arthur#/cards
[2]: https://splunk7.vtex.com/en-US/app/search/search?q=search%20index%3Dmy_account&display.page.search.mode=verbose&dispatch.sample_ratio=1&earliest=-5d%40d&latest=now&display.page.search.tab=events&sid=1565298680.90134_D318FDA5-14D0-4248-8853-FBCE1DBB4B96

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Menu/index.js [hunk 1]
```diff
@@ -5,11 +5,12 @@ import { ExtensionPoint } from 'render'
...
 class Menu extends Component {
   state = {
-    hasExtension: null
+    hasExtension: null,
   }
 
-  handleExtension = (hasExtension) => {
+  handleExtension = hasExtension => {
     this.setState({ hasExtension })
+    throw new Error('um erro de teste')
```
Comment (salesfelipe) [inline react/components/Menu/index.js]: Is this really necessary?

File: react/index.js [hunk 2]
```diff
@@ -1,17 +1,32 @@
-import React from 'react'
+import React, { Component } from 'react'
 import AppRouter from './components/AppRouter'
 import ClientSide from './components/ClientSide'
+import { logMyAccountURL, logGeneralErrors } from './SplunkUtils'
```
Comment (salesfelipe) [inline react/index.js]: Don't mix, relative and static imports.

File: react/components/Menu/index.js (hunk 1 above)
Comment (arthurepc) [reply react/components/Menu/index.js]: No, it wasn't. I removed it, thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Log in Splunk if URL that points to My Account is `/account/orders`.
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
- Log on Splunk if the URL that points to My Account is `/account/orders`.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Log if some error occurs, as a way of creating some deployment process.
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
- Log whenever some error occurs so we can create some deployment analysis process.
```

**Pull request merged**